e2e/fragments/dq/witnesses.page.js

```diff
@@ -28,14 +40,27 @@ module.exports = {
       I.click(this.fields(party).witnessesToAppear.options.yes);
     });
 
-    await this.addWitness(party);
+    let isHNLEnabled = await checkToggleEnabled('hearing-and-listing-sdo');
+    if (!isHNLEnabled || !(['preview', 'demo'].includes(config.runningEnv))) {
```

Comment:
Should this just be checking the feature toggle not the env? I think we want the old fields in preview at the moment since we exclude our new hnl changes in preview so it will be looking for the old fields.